test/testutils.js

```diff
@@ -110,6 +110,10 @@ function checkHeaders(res) {
     'X-Requested-With, Content-Type');
 }
 
+function dumpBody(res) {
+  console.log(res.body);
```

Comment:
I thought the body was a string, but it looks like superagent does automatically parse if the Content-Type is set: http://visionmedia.github.io/superagent/#response-properties